src/app/src/androidTest/java/br/com/italoromeiro/arctouchtest/robots/SearchRobot.java

```diff
@@ -0,0 +1,24 @@
+package br.com.italoromeiro.arctouchtest.robots;
+
+import br.com.italoromeiro.arctouchtest.R;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+/**
+ * Created by italo on 13/08/16.
+ */
+public class SearchRobot {
+    public SearchRobot where(String nameOfStreet) {
```

Comment:
> "To test this screen I will type WHERE I want to go.

Now I get it, kind of. I'd still rather call it `streetName()` (just like the `recipient()` and `amount()` examples from Jake Wharton's presentation) or something else instead of `where()`.

And you have a point: it makes sense to keep the methods separated since you might want to use them like that in other tests. I thought that might be the reason why you kept them like this.